api/internal/core/entity/entity.go

```diff
@@ -17,24 +17,30 @@
 package entity
 
 import (
+	"reflect"
 	"time"
 
 	"github.com/apisix/manager-api/internal/utils"
 )
 
 type BaseInfo struct {
-	ID         string `json:"id"`
-	CreateTime int64  `json:"create_time"`
-	UpdateTime int64  `json:"update_time"`
+	ID         interface{} `json:"id"`
```

Comment:
Why not using `json.Number`?

Reply:
Hi, @tokers  
we support id as any string not only number.

Reply:
Oops, i misunderstood it.